src/EStimLibrary/Core/SpatialModel/LocalizationData.cs

```diff
@@ -10,6 +11,12 @@
 public record LocalizationData(IEnumerable<int> AreasFullyContaining,
     IEnumerable<int> AreasPartiallyContaining)
 {
+    // Modify the properties using ToList() to create shallow copies during init
+    public IEnumerable<int> AreasFullyContaining { get; init; } =
```

Comment:
Was something failing before making this change? This definitely ensures deep copies, which is a good thing rather than shallow copies, but just want to make sure adding this init step is needed and not inherently done via base record construction